src/main/java/com/ita/if103java/ims/entity/Event.java

```diff
@@ -0,0 +1,115 @@
+package com.ita.if103java.ims.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+public class Event {
+    private Long id;
+    private String message;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
+    private ZonedDateTime date;
+    private Long accountId;
+    private Long warehouseId;
+    private Long authorId;
+    private EventType type;
+    private Long transactionID;
```

Comment:
Change that line to be the same to other fields
```java 
private Long transactionId;
```
Also rename it for `get / set` methods

Reply:
Got it, did it